code/datums/components/riding/riding.dm

```diff
@@ -68,7 +68,8 @@
 	RegisterSignal(parent, COMSIG_ATOM_DIR_CHANGE, .proc/vehicle_turned)
 	RegisterSignal(parent, COMSIG_MOVABLE_UNBUCKLE, .proc/vehicle_mob_unbuckle)
 	RegisterSignal(parent, COMSIG_MOVABLE_MOVED, .proc/vehicle_moved)
-	RegisterSignal(parent, COMSIG_MOVABLE_BUMP, .proc/vehicle_bump)
+	if(!isxeno(parent))
+		RegisterSignal(parent, COMSIG_MOVABLE_BUMP, .proc/vehicle_bump)
```

Comment:
As I told you in discord this isnt the correct fix, as the real fix is to fix this allowing doors to be opened in the first place